lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -1182,13 +1160,8 @@ private function getSQLForInsert() : string
      */
     private function getSQLForUpdate() : string
     {
-        $from = $this->sqlParts['from'];
-
-        if ($from['alias'] === null || $from['alias'] === $from['table']) {
-            $table = $from['table'];
-        } else {
-            $table = $from['table'] . ' ' . $from['alias'];
-        }
+        /** @var string $table */
+        $table = $this->sqlParts['table'];
```

Comment:
This is no longer needed as an individual variable (the same in the rest of the cases).

Reply:
I'm surprised that phpstan doesn't complain about the unknown type converted to string, but it's OK: https://github.com/doctrine/dbal/pull/3833/commits/6adfcd5c3ecdc888bee32da7636d279832981d4f